src/app/news/news-list.component.ts

```diff
@@ -41,9 +42,23 @@ export class NewsListComponent implements OnChanges {
     private dialogsFormService: DialogsFormService,
     private dialogsLoadingService: DialogsLoadingService,
     private newsService: NewsService,
-    private planetMessageService: PlanetMessageService
+    private planetMessageService: PlanetMessageService,
+    private router: Router,
+    private route: ActivatedRoute
   ) {}
 
+  ngOnInit() {
+    const childRoute = this.route.firstChild;
+    if (childRoute) {
+      const voiceId = childRoute.snapshot.paramMap.get('id');
+      if (voiceId) {
+        this.newsService.requestNewsItem(voiceId).subscribe(news => {
```

Comment:
We already fetch all of the news on the CommunityComponent. That's sent to this component in `items` so you can just find it there rather than making another request.